ocis-pkg/oidc/metadata.go

```diff
@@ -53,6 +54,43 @@ type ProviderMetadata struct {
 	//claim_types_supported
 }
 
+// Logout Token
+type LogoutToken struct {
```

Comment:
comment of exported type `LogoutToken` should be of form `LogoutToken ...` (with an optional leading article)